src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+
+Using configurations, you can change this behavior. For example, you can define selectors to narrow or widen the selection of pods that are injected, assign resources to the operator, and tune the sidecar. Also, you can add other attributes, labels, and environment variables. Please refer to the chart `README.md` and `values.yaml` for the complete description of each option.
+
+<Callout variant="tip">
+Injecting the sidecar into deployments scheduled into EC2 nodes may lead into incorrect or duplicate data, especially if those nodes are already being monitored with the DaemonSet. Take this into account when designing custom selectors for the automatic injection.
+</Callout>
+
+#### Update to the latest version or to a new configuration
+
+To update to the latest version of the EKS Fargate integration, upgrade the chart and reinstall the bundle by simply running again the command above.
+
+To update the configuration of the infrastructure agent injected or the `newrelic-infra-operator` itself, modify the `values.yaml` and upgrade the Helm release with the new configuration. The `newrelic-infra-operator` is immediately updated; however, another rollout of the workloads is needed in order to inject the infrastructure agent sidecar with the new configuration.
+
+```
+  $ kubectl rollout restart deployment <var>YOUR_DEPLOY</var>
+```
+
+#### Uninstall the Fargate integration
+
+We recommend that you uninstall the infrastructure operator that performs the automatic injection but keep the rest of the New Relic solution. Using Helm, disable the infra-operator by setting `infra-operator.enabled` to `false`, either in the `values.yaml` file or in the command line `--set`, and re-run the upgrade `--install` command above.
+
+To uninstall the whole solution:
+
+1. Completely uninstall the Helm release.
+2. Rollout the pods in order to remove the sidecar:
+    ```
+    $ kubectl rollout restart deployment <var>YOUR_APP</var>
+    ```
+3. Garbage collect the secrets:
+    ```
+    $ kubectl delete secrets -n <var>YOUR_NAMESPACE</var> -l newrelic/infra-operator-created=true
+    ```
+#### Known limitations: automatic injection
+
+Here are some issues to be aware of when using automatic injection:
+
+1. Currently there is no controller that watches the whole cluster to make sure that secrets that are no longer needed are garbage collected. However, all objects share the same label that can be used in order to remove all resources if needed `newrelic/infra-operator-created: true`.
+
+2. At the moment, it is not possible to use the injected sidecar to monitor services running in the pod. The sidecar will only monitor Kubernetes itself. However, if you are an advanced user, you may want to exclude these pods from automatic injection. You would then manually inject a customized version of the sidecar with on-host integrations enabled by configuring them and mounting their configurations in the proper place. [MISSING LINK].
```

Comment:
https://docs.newrelic.com/docs/integrations/kubernetes-integration/link-apps-services/tutorial-monitor-redis-running-kubernetes/

Note that it is not a guide they can follow in this scenario, however an advanced user can use that to gather the required informations

Reply:
@paologallinaharbur, I will insert this link.